helm-chart-sources/pulsar/templates/broker-deployment/broker-configmap.yaml

```diff
@@ -58,6 +58,9 @@ data:
     {{- end }}
     {{- end }}
   clusterName: {{ template "pulsar.fullname" . }}
+  {{- if .Values.broker.brokerDeleteInactiveTopicsEnabled }}
+  brokerDeleteInactiveTopicsEnabled: {{ .Values.brokerDeleteInactiveTopicsEnabled }}
```

Comment:
This doesn't match the if above. `.Values.broker.brokerDeleteInactiveTopicsEnabled` vs `.Values.brokerDeleteInactiveTopicsEnabled` I think having the setting under `broker` is the right place for it.

Reply:
Fix the broker config map and add a default in the values.yaml.